hypothesis-python/tests/nocover/test_build_signature.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis/
+#
+# Most of this work is copyright (C) 2013-2020 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from inspect import signature
+
+from hypothesis import given, strategies as st
+
+
+def use_this_signature(*, x: int, y: str):
```

Comment:
Is there a reason why it isn't worthwhile to be complete and include all variety of arguments in this test?

Reply:
No point testing `*args` or `**kwargs` since they're never inferred, but I've added positional args with and without a default value.